recipes/mosdepth/build.sh

```diff
@@ -12,7 +12,8 @@ if [[ ${target_platform}  == osx-64 ]] ; then
     nimble install -y "docopt@0.7.0"
     nimble build -y --verbose -d:release
```

Comment:
can you add `-d:d4` on this line so that both OSX and linux will have d4?

Reply:
Good point! Done! 👍🏻